eth/db/journal.py

```diff
@@ -20,8 +20,18 @@ class DeletedEntry:
     pass
 
 
+# Track two different kinds of deletion:
+
+# 1. key in wrapped
+# 2. key modified in journal
+# 3. key deleted
 DELETED_ENTRY = DeletedEntry()
 
+# 1. key not in wrapped
+# 2. key created in journal
+# 3. key deleted
+ERASE_CREATED_ENTRY = DeletedEntry()
+
 
 class Journal(BaseDB):
     """
```

Comment:
nitpick: 100% not part of this PR, but the docstring on `is_flattened` is wrong. If you're feeling generous now is as good a time as any to fix it :)